include/slang.h

```diff
@@ -4687,6 +4687,66 @@ struct SlangGlobalSessionDesc
     uint32_t reserved[16] = {};
 };
 
+/* Create a blob from binary data.
+ *
+ * @param data Pointer to the binary data to store in the blob. Must not be null.
+ * @param size Size of the data in bytes. Must be greater than 0.
+ * @return The created blob on success, or nullptr on failure.
+ */
+SLANG_EXTERN_C SLANG_API ISlangBlob* slang_createBlob(const void* data, size_t size);
```

Comment:
The API should be `SlangResult slang_createBlob(const void* data, size_t size, ISlangBlob** outBlob);`